addon/mixins/touch-action.js

```diff
@@ -37,15 +36,15 @@ export default Mixin.create({
   ignoreTouchAction: false,
 
   init() {
-    this._super();
+    this._super(...arguments);
 
     const {
       tagName,
       ignoreTouchAction,
       click
     } = this;
 
-    const hasClick = click !== K;
+    const hasClick = typeof click === 'function';
```

Comment:
yes - i agree and have come across the argument for not using the typeof check - one question - given Ember.K returns `this`, is it possible for a user to return `this` from their click handler as well and have the dirty reference check fail?

also I though just checking via `if (this.click)` works - that doesn't de-opt in chrome right?